packages/codegen-ui-react/lib/forms/form-renderer-helper/bidirectional-relationship.ts

```diff
@@ -118,7 +228,18 @@ export function getBiDirectionalRelationshipStatements({
       ? relationship.associatedFields
       : [];
 
+  const isFieldRequired = dataSchema.models[modelName].fields[fieldName].required;
+  const isRelatedModelRequired = fieldBiDirectionalWith.required;
```

Comment:
May I suggest this variable be renamed to `isFieldBiDirectionalWithRequired`--`isRelatedModelRequired` does not quite capture the meaning

Reply:
Yeah that looks better.